server.js

```diff
@@ -1,24 +1,137 @@
-/*jshint esversion: 6 */
+      /*jshint esversion: 6 */
 
-const express = require('express');
-const router = express.Router();
-const app = express();
-const PORT = process.env.PORT || 9000;
-const bodyParser = require('body-parser');
-const imageRouter = require('./routes/image_routes');
+      const express = require('express');
+      const router = express.Router();
+      const app = express();
+      const PORT = process.env.PORT || 9000;
+      const bodyParser = require('body-parser');
+      const imageRouter = require('./routes/image_routes');
+      const methodOverride = require('method-override');
 
-app.use(bodyParser.urlencoded({extended: true}));
-app.use('/gallery', imageRouter);
+      const passport = require('passport');
+      const LocalStrategy = require('passport-local').Strategy;
+      const session = require('express-session');
 
-const db = require('./models');
+      const RedisStore = require('connect-redis')(session);
+      const saltRounds = 10;
+      const bcrypt = require('bcrypt');
 
-const Images = db.Images;
-const Authors = db.Authors;
+      const expressHb = require('express-handlebars');
+      const hbs = expressHb.create({
+        defaultLayout: 'main',
+        extname:'hbs'
+      });
 
+      const db = require('./models');
 
+      const Images = db.Images;
+      const Authors = db.Authors;
+      const Users = db.Users;
 
-app.listen(PORT, ()=>{
-  //db.sequelize.drop();
-  db.sequelize.sync();
-  console.log(`server running on ${PORT}`);
-});
\ No newline at end of file
+      app.engine('hbs', hbs.engine);
+      app.set('view engine', 'hbs');
+
+
+      app.use(express.static('public'));
+
+
+      app.use(bodyParser.urlencoded({extended: true}));
+
+
+      app.use(methodOverride('_method'));
+
+
+      app.use(session({
+        store: new RedisStore(),
+        secret: 'keyboard cat',
+        resave: false,
+        saveUninitiazlied: false
+      }));
+
+
+      app.use(passport.initialize());
+      app.use(passport.session());
+
+      passport.use(new LocalStrategy((username, password, done) => {
+        return Users.findAll({where: {username: username}})
```

Comment:
`Users.find` would be better here, then you can check `results !== null` instead of length.